Data-Structure-Stacks.md

```diff
@@ -0,0 +1,60 @@
+# Data Structure Stacks
+
+`Stack` is a linear data structure which follows a particular order in which the operations are performed. The order may be 
+LIFO(Last In First Out) or FILO(First In Last Out).
```

Comment:
I also didn't found any code examples for this. But this concept is applicable to the stack when we keep popping elements from the stack then the last element which we would get will the element which was first pushed to the stack.